app/controllers/projects_controller.rb

```diff
@@ -175,39 +175,11 @@ def set_projects_query
       @applied_filters = {}
 
       @projects = Project
-      @projects = @projects.tagged_with(params[:skills], any: true, on: :skills) if params[:skills].present?
+      @projects = @projects.where(needs_funding: params[:needs_funding]) if params[:needs_funding].present?
       @projects = @projects.tagged_with(params[:project_types], any: true, on: :project_types) if params[:project_types].present?
-      @projects = @projects.tagged_with(params[:for_profit_type], any: true, on: :for_profit_type) if params[:for_profit_type].present?
-      @projects = @projects.where(accepting_volunteers: params[:accepting_volunteers] == '1') if params[:accepting_volunteers].present?
-      @projects = @projects.where(highlight: true) if params[:highlight].present?
-      @projects = @projects.where(target_country: params[:target_country]) if params[:target_country].present?
-      @projects = @projects.where(status: params[:status]) if params[:status].present?
-
-      if params[:query].present?
-        @projects = @projects.search(params[:query]).left_joins(:volunteers).reorder(nil).group(:id)
-      else
-        @projects = @projects.left_joins(:volunteers).group(:id)
-      end
-
-      if params[:sort_by]
-        @projects = @projects.order(get_order_param)
-      else
-        @projects = @projects.order('highlight DESC, COUNT(volunteers.id) DESC, created_at DESC')
-      end
-
-      if params[:project_types].present?
-        @applied_filters[:project_types] = params[:project_types]
-      end
-
-      if params[:for_profit_type].present?
-        @applied_filters[:for_profit_type] = params[:for_profit_type]
-      end
-
-      if params[:skills].present?
-        @applied_filters[:skills] = params[:skills]
-      end
 
-      @projects = @projects.includes(:project_types, :skills, :volunteers)
+      @projects = @projects.includes(:project_types)
```

Comment:
I inititally wanted to delete this line, as for the moment I don't need to 'include' this extra information for every project/business. But when I remove it, [this test fails](https://github.com/helpwithcovid/covid-volunteers/blob/bf1d98d48a5ea8bf469f85391d936ebd716e8d11/spec/controllers/projects_controller_spec.rb#L17) with this error:
```
  1) ProjectsController GET #index returns json
     Failure/Error: render json: @projects

     SystemStackError:
       stack level too deep
     # ./app/controllers/projects_controller.rb:48:in `block (2 levels) in index'
     # ./app/controllers/projects_controller.rb:35:in `index'
     # ./app/controllers/application_controller.rb:119:in `switch_locale'
     # ./spec/controllers/projects_controller_spec.rb:18:in `block (3 levels) in <top (required)>'
```
Any ideas on why this is happening?